internal/pkg/telemetry/ddhttp.go

```diff
@@ -0,0 +1,117 @@
+package telemetry
+
+import (
+	"context"
+	"net/http"
+	"reflect"
+	"strings"
+	"time"
+
+	"github.com/keboola/go-client/pkg/client"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	ddtracer "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/strhelper"
+)
+
+func HttpClientTraceFactory() client.TraceFactory {
```

Comment:
`HTTPClientTraceFactory`? 😄

Reply:
Ako to mam nazvat? :slightly_smiling_face:  `DDTraceFactory` znie lepsie

Reply:
https://github.com/keboola/keboola-as-code/pull/999/commits/902332719643821b05aee515c4a662d084d0b72a